### PR TITLE
Add policy definition

### PR DESCRIPTION
**Pull request by vivsriaus**

### CODE REVIEW

File: arm-resources/2014-04-01-preview/swagger/resources.json [hunk 1]
```diff
@@ -1508,9 +1508,137 @@
...
+          "$ref": "#/definitions/PolicyDefinitionProperties",
+          "description": "Gets or sets the policy definition properties."
+        },
+        "name": {
+          "type": "string",
+          "description": "Gets or sets the policy definition name."
+        }
+      },
+        "description": "Policy definition."
+      },
```
Comment (stankovski) [inline arm-resources/2014-04-01-preview/swagger/resources.json]: Do you want properties to be "flattened"? If so, apply `"x-ms-azure-resource": "true"` extension

File: arm-resources/2014-04-01-preview/swagger/resources.json [hunk 2]
```diff
@@ -1508,9 +1508,137 @@
           "nextLinkName": null
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Authorization/policydefinitions/{policyDefinitionName}": {
+      "put": {
```
Comment (stankovski) [inline arm-resources/2014-04-01-preview/swagger/resources.json]: Is this a LRO operation?

Comment (vivsriaus) [inline arm-resources/2014-04-01-preview/swagger/resources.json]: This isn't a LRO operation

File: arm-resources/2014-04-01-preview/swagger/resources.json (hunk 1 above)
Comment (vivsriaus) [inline arm-resources/2014-04-01-preview/swagger/resources.json]: @stankovski What effect does this have on the generated code?

Comment (vivsriaus) [inline arm-resources/2014-04-01-preview/swagger/resources.json]: Done, thanks for the explanation

**Pull request merged**